frontend/app/controllers/Application.scala

```diff
@@ -13,12 +20,32 @@ object Application extends Controller {
     Ok(views.html.index("hello"))
   }
 
+  def quote = WebSocket.acceptWithActor[String, String] { request =>
+    out => 
+      Props(classOf[TsMsgToJson[Quote]], out, implicitly[ClassTag[Quote]])
+  }
+  
+
+  def transaction = WebSocket.acceptWithActor[String, String] { request =>
+    out => 
+      Props(classOf[TsMsgToJson[Transaction]], out, implicitly[ClassTag[Transaction]])
+  }
+
+  def ohlc = WebSocket.acceptWithActor[String, String] { request =>
+    out => 
+      Props(classOf[TsMsgToJson[OHLC]], out, implicitly[ClassTag[OHLC]])
+  }
+
+  def sma = WebSocket.acceptWithActor[String, String] { request =>
+    out => 
+      Props(classOf[TsMsgToJson[SMA]], out, implicitly[ClassTag[SMA]])
```

Comment:
I don't think you will get these messages. Let's simply not use them in the frontend.